src/jit/codegencommon.cpp

```diff
@@ -3466,7 +3460,7 @@ void CodeGen::genFnPrologCalleeRegArgs(regNumber xtraReg, bool* pXtraRegClobbere
             }
         }
 
-        var_types regType = varDsc->TypeGet();
+        var_types regType = compiler->mangleVarArgsType(varDsc->TypeGet());
```

Comment:
Should regArgTab store the manged type, to avoid re-calling mangeVarArgsType?

Reply:
>Maybe I haven't read far enough, but I don't understand this comment. It was done as is because SoftFP should behave exactly as varargs, where everything gets passed in integer registers on arm(el). Is that not true anymore?

What you are saying is correct, if you are referring to the windows arm vararg ABI. For the unix vararg implementations that is incorrect.

Reply:
I will leave this for a later change.